src/components/InfoPanel.component.js

```diff
@@ -32,6 +37,29 @@ class InfoPanel extends Component {
     const isShownNext = curIndex < activeEvents.length - 1
     const isShownPrev = curIndex > 0
 
+    const carouselInstance = (
+      <Carousel>
+        <Carousel.Item>
+          <img width={900} height={500} alt="900x500" src={IMAGE_URL_1} />
```

Comment:
Maybe use a react bootstrap `Image` component. Also, the `alt` prop is used for text that appears in case the image doesn't load (among other things). So think about what part of the poi content could be used instead